client/signup/steps/domains/index.jsx

```diff
@@ -88,7 +98,7 @@ module.exports = React.createClass( {
 				} ) :
 				undefined;
 
-		SignupActions.submitSignupStep( {
+		SignupActions.submitSignupStep( Object.assign( {}, {
```

Comment:
Yep, you're right, I think that's almost muscle memory now (not necessarily a bad thing ;) ). Changed it.